File: Client/CredentialPermissionService.cs
using KolibSoft.AuthStore.Core.Filters;
using KolibSoft.AuthStore.Core.Models;
using KolibSoft.Catalogue.Client;

namespace KolibSoft.AuthStore.Client;

public class CredentialPermissionService : CatalogueService<CredentialPermissionModel, CredentialPermissionFilters>
{
    public CredentialPermissionService(HttpClient httpClient, string uri) : base(httpClient, uri) { }
}

File: docs/lib/services/credential_permission_service.js
import { CredentialPermissionModel } from "../models/credential_permission_model.js";
import { CatalogueService } from "../modules/catalogue.js"

class CredentialPermissionService extends CatalogueService {

    /**
     * @param {(json: {}) => CredentialPermissionModel} creator 
     * @param {(input: RequestInfo | URL, init?: RequestInit | undefined) => Promise<Response>} fetch 
     * @param {string} uri 
     */
    constructor(creator, fetch, uri) {
        super(creator, fetch, uri);
    }

}

export {
    CredentialPermissionService
}